text/0000-multiarch-builders-and-package.md

```diff
@@ -0,0 +1,1090 @@
+                │    ├── build
+                │    └── detect
+                ├── buildpack.toml
+                └── foo
+```
+
+#### Buildpacks authors do not use targets or the new folder structure
+
+This seems to be the case for [Paketo Buildpacks](https://github.com/paketo-buildpacks/maven) 
+or [Heroku](https://github.com/heroku/buildpacks-jvm/tree/main/buildpacks/maven), and it represents how `pack` 
+will work for most the users when new behavior is implemented
+
+A simplified version of Buildpack Authors folder structures is:
+
+```bash
+├── bin
+│ ├── build
+│ └── detect
+├── buildpack.toml
+└── package.toml
+```
+
+In these cases: We expect `pack` to keep doing what is doing today, but with the warning messages we mentioned above to 
+let end users know things are changing. 
+
+```bash
+pack buildpack package <buildpack> --config ./package.toml --publish 
+Warning: A new '--target' flag is available to set the platform for the buildpack package, using 'linux' as default
+Successfully published package <buildpack> and saved to registry
+
+# Or
+pack buildpack package  <buildpack> --config ./package.toml --format file 
+Warning: A new '--target' flag is available to set the platform for the buildpack package, using 'linux' as default
+Successfully created package <buildpack> and saved to file
+ ```
+**Output**: pack will create a buildpack package image (as it is doing it today) with the provided binaries and a 
+[configuration](https://github.com/opencontainers/image-spec/blob/main/config.md#properties) with the following target 
+platform:
+
+```json
+{
+  "architecture": "",
+  "os": "linux"
+}
+```
+
+After checking the **warning** messages, some end users must feel curious, and the try to use the new `--target` flag.
+
+```bash
+pack buildpack package <buildpack> --config ./package.toml --publish --target linux/arm64
+Successfully published package <buildpack> and saved to registry
+
+# Or
+pack buildpack package  <buildpack> --config ./package.toml --format file --target linux/arm64
+Successfully created package <buildpack> and saved to file
+```
+
+**Output**: In these cases, pack will create buildpack a package image with the provided binaries and a 
+[configuration](https://github.com/opencontainers/image-spec/blob/main/config.md#properties) with the following target 
+platform:
+
+```json
+{
+  "architecture": "arm64",
+  "os": "linux"
+}
+```
+
+> **Important**
+> Pack will assume the binaries are appropriate for the given target platform, what the flag is doing is expose a mechanism
+> to update the metadata present in the OCI config file
+
+what about creating a multi-arch image for several target platforms?
+
+```bash
+pack buildpack package <buildpack> --config ./package.toml --publish --target linux/arm64 --target linux/amd64
+A multi-arch buildpack package will be created for target platforms: 'linux/amd64', 'linux/arm64'
+Successfully published package <buildpack> and saved to registry
```

Comment:
```suggestion
Successfully published <buildpack>
```

Reply:
This is actually the current messaged showed in pack